getnacelle/nacelle-launch-tests
# Update Shopify Accounts example to use serverless functions for multipass and countrycitstatejson

### WHY are these changes introduced?

Addresses [[DRAMS-223](https://nacelle.atlassian.net/browse/DRAMS-223)]

> Update Nacelle Shopify Accounts example and associated docs to abstract heavy packages to cloud functions

### WHAT is this pull request doing?

- updates the nuxt-shopify-accounts project folder to use serverless functions for `multipassify` and `countrycitystatejson`
- serverless functions are provided for both Vercel and Netlify
- updates the Nuxt project code to match the latest Nuxt starter, but subtracts ALL parts not needed to demonstrate how the accounts work. Styles and components are quite minimal
- updates to the Readme -- see those changes for more details

#### Updated from @NWRichmond  in #24 
- When the user initiates checkout, redirects the user to the Multipassify URL, which returns the user to the Shopify checkout URL
- Adds a single item to the cart to facilitate checkout without adding more data plumbing or components
- Renames Shopify token environment variable names to make it obvious which values need to be retrieved from the Shopify Admin
- Ditches `axios` except for in Serverless Functions in order to steer merchant developers towards lightweight http clients like `isomorphic-unfetch`

### How to Test

Pull down the branch and run locally with either Vercel CLI or Netlify CLI -- see npm scripts in package.json
